R/backend-mssql.R

```diff
@@ -107,6 +107,7 @@ simulate_mssql <- function(version = "15.0") {
                                                     conflict = c("error", "ignore"),
                                                     returning_cols = NULL,
                                                     method = NULL) {
+  check_string(method, allow_null = TRUE)
```

Comment:
IMO the order of argument checks should match the order of the arguments.

Also can we say anything about the type of `returning_cols`?